src/lib/storage/column_encoders/column_encoder.template.hpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * Column Encoder Template
+ *
+ * - Copy this file for new column encoders
+ * - Add new encoder to encoders.hpp
+ */
+#pragma once
```

Comment:
Is this compiled and tested somewhere? Code that is not part of the program *will* become outdated.

Reply:
This file should be updated if the interface changes. They is a similar already existing file:`test/gtest_case_template.cpp`. You are of course right, it is easy to forgot updating this file. Shall I remove it?